### ISSUE TITLE
Problems with iOS8?

### ISSUE THREAD

**Issue by josephmisiti** [danielebogo/DBCamera#186]
I am seeing a `EXC_BAD_ACCESS` on iphone 6 running iOS 8.3:

```
            err = CVOpenGLESTextureCacheCreateTextureFromImage (kCFAllocatorDefault, coreVideoTextureCache, renderTarget,
                                                                NULL, // texture attributes
                                                                GL_TEXTURE_2D,
                                                                _textureOptions.internalFormat, // opengl format
                                                                (int)_size.width,
                                                                (int)_size.height,
                                                                _textureOptions.format, // native iOS format
                                                                _textureOptions.type,
                                                                0,
                                                                &renderTexture);

```

**Comment by josephmisiti** [comment]
same problem here: http://stackoverflow.com/questions/31035059/cvopenglestexturecachecreatetexturefromimage-error

**Comment by josephmisiti** [comment]
It looks like it might be a bug in iOS 8.3:

https://github.com/BradLarson/GPUImage/issues/77#issuecomment-121289985

**Comment by josephmisiti** [comment]
I can confirm this issue is resolved in iOS 8.4. Closing ...
